src/dotnet/RiderPlugin.UnrealLink/PluginInstaller/UnrealPluginDetector.cs

```diff
@@ -90,57 +91,33 @@ public class UnrealPluginDetector
                                     });
                                 return;
                             }
-
+                            
+                            var riderLinkFolders = myProjectsTracker.GetAllPlugins().Where(pluginPath => pluginPath.NameWithoutExtension.Equals("RiderLink")).ToList();
+                            // foreach (var pluginRoot in riderLinkFolders)
```

Comment:
Commented code looks suspisious. Is it forgotten or is it not needed at all?

Reply:
The whole PR is bad, I've missclicked the branch, my bad, give me a sec